src/core/config/openthread-core-default-config.h

```diff
@@ -294,13 +294,13 @@
 #endif
 
 /**
- * @def OPENTHREAD_CONFIG_HEAP_EXTERNAL_ENABLE
+ * @def OPENTHREAD_CONFIG_EXTERNAL_HEAP_ENABLE
  *
  * Enable the external heap.
  *
  */
-#ifndef OPENTHREAD_CONFIG_HEAP_EXTERNAL_ENABLE
-#define OPENTHREAD_CONFIG_HEAP_EXTERNAL_ENABLE 0
```

Comment:
Should we add a checking in openthread-core-config-check.h to tell users that `OPENTHREAD_CONFIG_HEAP_EXTERNAL_ENABLE` is now replaced by `OPENTHREAD_CONFIG_EXTERNAL_HEAP_ENABLE`?

Reply:
Sure, thanks!